easybuild/easyconfigs/g/GCC/GCC-11.3.0.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'Bundle'
+
+name = 'GCC'
+version = '11.3.0'
+
+homepage = 'https://gcc.gnu.org/'
+description = """The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
+ as well as libraries for these languages (libstdc++, libgcj,...)."""
+
+toolchain = SYSTEM
+
+dependencies = [
+    ('GCCcore', version),
+    # binutils built on top of GCCcore, which was built on top of (dummy-built) binutils
```

Comment:
`s/(dummy-built) binutils/binutils (built with system toolchain)/g`

Reply:
Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/15311/commits/e2d3ee81d582f718dd86de717711310f2a0e5463